OSPSuite.Presentation.Importer/Views/IColumnMappingControl.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using OSPSuite.Assets;
+using OSPSuite.Presentation.Importer.Core.DataFormat;
+using OSPSuite.Presentation.Importer.Presenters;
+using OSPSuite.Presentation.Views;
+using OSPSuite.Utility.Reflection;
+
+namespace OSPSuite.Presentation.Importer.Views
+{
+   public class ColumnMappingViewModel : Notifier
```

Comment:
This should be in the presenter namespace

To fit our naming convention, call it DTO not ViewModel

Reply:
Done